src/mongocrypt-cache.c

```diff
@@ -242,6 +283,10 @@ _mongocrypt_cache_cleanup (_mongocrypt_cache_t *cache)
 {
    _mongocrypt_cache_pair_t *pair, *tmp;
 
+   if (!cache) {
+      return;
+   }
```

Comment:
```suggestion
   BSON_ASSERT_PARAM (cache);
```
Assert `cache` is not-null.